src/signalcorr.jl

```diff
@@ -56,6 +56,8 @@ before computing the autocovariance.
 If `x` is a vector, `r` must be a vector of the same length as `x`.
 If `x` is a matrix, `r` must be a matrix of size `(length(lags), size(x,2))`, and
 where each column in the result will correspond to a column in `x`.
+
+The output is not normalized by the variance. See `autocor!` for normalization including the variance.
```

Comment:
Could just say "is not normalized." (either by the variance of by anything). Also, it would be good to normalize (pun intended) the second sentence so that the phrasing is the same across docstrings.